src/main/kotlin/view/InputView.kt

```diff
@@ -0,0 +1,94 @@
+package view
+
+import model.Lotto
+import model.Ticket
+
+class InputView {
+    fun getUserInputAsString(
+        prompt: String,
+        numberOfAttempts: Int = 3,
+    ): String {
+        if (numberOfAttempts <= 0) {
+            throw IllegalArgumentException(ErrorMessages.INPUT_TOO_MANY_ATTEMPT.message) // check
+        }
+        println(prompt)
+        val input = readLine()?.trim()
```

Comment:
```suggestion
        val input = readlnOrNull()?.trim()
```
Let's find out what the differences are between these two.

Reply:
Thanks for pointing it out, I learned that readlnOrNull is more modern way/naming of using it!

# readLine()
- Return type: String?
- Since : Kotlin 1.0 (old version)
- When EOF : return `null` -> either handle it or use `!!` to force non-null

# readln()
- Return type: String
- Since: Kotlin 1.6.0
- When EOF: throw `EOFException` (kotlin.io.EOFException), never return null
- Exactly same as: `readlnOrNull()!!` 
-> but using `readlnOrNull()` is more graceful way
-> If you want to crash on EOF, just using `readln()` is enough

# readlnOrNull()
- return type: String?
- since: Kotlin 1.6.0
- When EOF: return `null`